broker/broker.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package broker
+
+import (
+	"context"
+
+	"github.com/nats-io/nats.go"
+)
+
+// SubjectAllChannels allows to subscribe to all subjects of all channels
+const SubjectAllChannels = "channel.>"
```

Comment:
if we have `channels/<ch_id>/messages` why this is not channels?

Reply:
@drasko had the same remark. But maybe we can fix this in another PR?

Reply:
sure,  just not to forget ti

Reply:
issue :)